composer.json

```
@@ -1,18 +1,18 @@
 {
-	"name": "wordpress/block-hydration-experiments",
+	"name": "wordpress/block-interactivity-experiments",
 	"type": "wordpress-plugin",
 	"license": "GPL-2.0-or-later",
-	"description": "Experiments with frontend hydration for blocks",
+	"description": "Experiments to add interactivity to blocks",
```

Maybe it makes sense to keep the word "frontend" here in some way?

```suggestion
	"description": "Experiments to add frontend interactivity to blocks",
```

or

```suggestion
	"description": "Experiments to add interactivity to blocks' frontends",
```